grails-app/controllers/org/pih/warehouse/api/CombinedShipmentItemApiController.groovy

```diff
@@ -112,23 +114,29 @@ class CombinedShipmentItemApiController {
         render (status: 200, text: "Successfully imported template")
     }
 
+    ApplicationTagLib getApplicationTagLib() {
+        return Holders.grailsApplication.mainContext.getBean(ApplicationTagLib)
+    }
+
     def exportTemplate() {
         def sw = new StringWriter()
 
+        ApplicationTagLib g = applicationTagLib
```

Comment:
Is there a reason for that change? I prefer using applicationTagLib instead of g in the code below.

Reply:
yeah I generally try to avoid abbreviated variable names to avoid confusion

Reply:
yeah that makes sense to be consistent with what our gsp files are doing. Do you know if that `<g:..>` is something we started or if it's a built in grails thing?

Reply:
@EWaterman it's a built in grails thing. 
So are `<g:if>`, `<g:hasErrors>` etc.